molgenis/molgenis-r-armadillo
# fix: subset

## To test
- Checkout branch
- run `devtools::load_all()`
- run `devtools::check()`
- Code review
- run this code, logging in using urls with and without the slash:

```
url <- "https://armadillo-demo.molgenis.net"
armadillo.login(url)
vars_to_subset <- c("child_id", "recruit_age", "coh_country", "mother_id", "preg_no", "child_no", 
                    "cob_m", "ethn1_m", "ethn2_m", "ethn3_m")

armadillo.subset(
  input_source = "arguments",
  source_project = "lifecycle",
  source_folder = "core", 
  source_table = "nonrep",
  target_project = "fix-subset", 
  target_folder = "core", 
  target_table = "testtable", 
  target_vars = vars_to_subset)

armadillo.delete_project("fix-subset")

url <- "https://armadillo-demo.molgenis.net/"
armadillo.login(url)

armadillo.subset(
  input_source = "arguments",
  source_project = "lifecycle",
  source_folder = "core", 
  source_table = "nonrep",
  target_project = "fix-subset", 
  target_folder = "core", 
  target_table = "testtable", 
  target_vars = vars_to_subset)
```
The view should work correctly in both cases.
